node/cardano-sl.cabal

```diff
@@ -25,6 +25,7 @@ Flag with-explorer
   description: Build with explorer
 
 library
+  extra-libraries:      m
```

Comment:
Our used nixpkgs commit a905b7cd0c2dc0714195a50bf176cd8e4593502d includes the GHC patch in aafe3d29c1572b89aa0f4f8095ffd9a43c77229e - so I wonder why we need this?

Reply:
Also once we figure out why we need this, it will have to be enabled only for linux